8-web-components/6-shadow-dom-style/article.md

```diff
@@ -311,24 +311,24 @@ customElements.define('user-card', class extends HTMLElement {
 
 
 
-## Summary
+## Resumen
 
-Shadow DOM can include styles, such as `<style>` or `<link rel="stylesheet">`.
+Shadow DOM puede incluir estilos, como `<style>` o `<link rel="stylesheet">`.
 
-Local styles can affect:
+Los estilos locales pueden afectar:
 - shadow tree,
-- shadow host with `:host`-family pseudoclasses,
-- slotted elements (coming from light DOM), `::slotted(selector)` allows to select  slotted elements themselves, but not their children.
+- shadow host con `:host`-pseudoclases familiares,
+- elementos slotted (provenientes de light DOM), `::slotted(selector)` permite seleccionar elementos slotted, pero no a sus hijos.
 
-Document styles can affect:
-- shadow host (as it lives in the outer document)
-- slotted elements and their contents (as that's also in the outer document)
+Los estilos de documentos pueden afectar:
+- shadow host (ya que vive en el documento exterior)
```

Comment:
DUDA

documento exterior o 
fuera del documento?

Reply:
Pienso que lo mas acertado es **documento exterior**, **fuera del documento** no tendria sentido en este contexto.

Reply:
Es lo que me falta conceptualmente...
pero... hay OTRO document? 
o es que el shadow está dentro... pero justamente dice que esta fuera...

Para entenderlo deberia trabajar con esto,
deberia abandonar la review que la tome otro.

ayudo con ortografia jaja

Reply:
Segun el ejemplo completo en la parte de **CSS hooks with custom properties** con outer document hace referencia al documento mas externo, es la parte donde declara la variable(el exterior).

For example, in shadow DOM we can use --user-card-field-color CSS variable to style fields, and the outer document can set its value:

<style>
  user-card {
    --user-card-field-color: green;
  }
</style>

Luego lo que esta dentro del template, lo refiere como algo mas interior o anidado en el documento
<template id="tmpl">
  <style>
*!*
    .field {
      color: var(--user-card-field-color, black);
    }
*/!*
  </style>
  <div class="field">Name: <slot name="username"></slot></div>
  <div class="field">Birthday: <slot name="birthday"></slot></div>
</template>

Si te fijas, segun la definición de outer segun https://dictionary.cambridge.org/es/diccionario/ingles/outer
refiere a esto: **on the outside, or at a greater distance from the center or inside**.

Reply:
!!! 👍 👍 👍 
outer puede ser fronterizo!
thanks for the schooling